src/components/sprite-info/sprite-info.jsx

```diff
@@ -3,14 +3,18 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 import Box from '../box/box.jsx';
-import BufferedInput from '../buffered-input/buffered-input.jsx';
+import Label from '../forms/label.jsx';
+import Input from '../forms/input.jsx';
+import BufferedInputHOC from '../forms/buffered-input-hoc.jsx';
+
 import styles from './sprite-info.css';
 
 import xIcon from './icon--x.svg';
 import yIcon from './icon--y.svg';
 import showIcon from './icon--show.svg';
 import hideIcon from './icon--hide.svg';
 
+const BufferedInput = BufferedInputHOC(Input);
```

Comment:
We could but it would make the import component dependent on the hoc, which seems wrong. I think a different file might be warranted for common combinations of stuff from forms.